lib/Doctrine/DBAL/Migrations/Tools/Console/Command/DiffCommand.php

```diff
@@ -109,7 +109,7 @@ public function execute(InputInterface $input, OutputInterface $output)
         }
 
         $version = date('YmdHis');
-        $path = $this->generateMigration($configuration, $input, $version, $up, $down);
+        $path = $this->generateMigration($configuration, $input, $version, $up, $down, true);
```

Comment:
I understand.  However, it'd be better to not add to the problems and instead use best-practices.  The concerns Fowler has about this style are valid regardless of the current state of the code.